### PR TITLE
Adding Cisco CVEs for mid Sep 2018 Publications

### PR DESCRIPTION
**Pull request by santosomar**
Adding 32 Cisco CVEs to the repository.

### CODE REVIEW

**Comment by santosomar** [comment]
Sent an email to cve@mitre.org for assistance. Thank you!

File: 2018/0xxx/CVE-2018-0432.json [hunk 1]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0432.json]: u'version' is a required property
[u'Cisco SD-WAN Solution '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0431.json [hunk 2]
```diff
@@ -1,18 +1,79 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0431.json]: u'version' is a required property
[u'Cisco Unified Computing System E-Series Software (UCSE) ', u'Cisco Unified Computing System (Management Software) ', u'Cisco 5000 Series Enterprise Network Compute System '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0437.json [hunk 3]
```diff
@@ -1,18 +1,78 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0437.json]: u'version' is a required property
[u'Cisco Umbrella ', u'Cisco Umbrella Roaming Module for Cisco AnyConnect ', u'Cisco Umbrella Enterprise Roaming Client for Mac ', u'Cisco Umbrella Enterprise Roaming Client for Windows ', u'Cisco Umbrella Enterprise Roaming Client for MacOS '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0436.json [hunk 4]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0436.json]: u'version' is a required property
[u'Cisco Webex Teams '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0460.json [hunk 5]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0460.json]: u'version' is a required property
[u'Cisco Enterprise NFV Infrastructure Software '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0423.json [hunk 6]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0423.json]: u'version' is a required property
[u'Cisco RV130W Wireless-N Multifunction VPN Router Firmware ', u'Cisco RV110W Wireless-N VPN Firewall Firmware ', u'Cisco RV215W Wireless-N VPN Router Firmware '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0424.json [hunk 7]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0424.json]: u'version' is a required property
[u'Cisco RV130W Wireless-N Multifunction VPN Router Firmware ', u'Cisco RV110W Wireless-N VPN Firewall Firmware ', u'Cisco RV215W Wireless-N VPN Router Firmware '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0422.json [hunk 8]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0422.json]: u'version' is a required property
[u'Cisco WebEx Meeting Center ', u'Cisco WebEx Meetings Server ', u'Cisco WebEx Meetings '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0447.json [hunk 9]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0447.json]: u'version' is a required property
[u'Cisco Email Security Appliance (ESA) '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0457.json [hunk 10]
```diff
@@ -1,18 +1,75 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0457.json]: u'version' is a required property
[u'Cisco WebEx WRF Player ', u'Cisco WebEx ARF Player '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0462.json [hunk 11]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0462.json]: u'version' is a required property
[u'Cisco Enterprise NFV Infrastructure Software '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0451.json [hunk 12]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0451.json]: u'version' is a required property
[u'Cisco Tetration Analytics '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0439.json [hunk 13]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0439.json]: u'version' is a required property
[u'Cisco Meeting Server '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0445.json [hunk 14]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0445.json]: u'version' is a required property
[u'Cisco Packaged Contact Center Enterprise '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0425.json [hunk 15]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0425.json]: u'version' is a required property
[u'Cisco RV130W Wireless-N Multifunction VPN Router Firmware ', u'Cisco RV110W Wireless-N VPN Firewall Firmware ', u'Cisco RV215W Wireless-N VPN Router Firmware '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0426.json [hunk 16]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0426.json]: u'version' is a required property
[u'Cisco RV130W Wireless-N Multifunction VPN Router Firmware ', u'Cisco RV110W Wireless-N VPN Firewall Firmware ', u'Cisco RV215W Wireless-N VPN Router Firmware '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0450.json [hunk 17]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0450.json]: u'version' is a required property
[u'Cisco Data Center Network Manager '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0414.json [hunk 18]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0414.json]: u'version' is a required property
[u'Cisco Secure Access Control Server Solution Engine (ACSE) '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0430.json [hunk 19]
```diff
@@ -1,18 +1,79 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0430.json]: u'version' is a required property
[u'Cisco Unified Computing System E-Series Software (UCSE) ', u'Cisco Unified Computing System (Management Software) ', u'Cisco 5000 Series Enterprise Network Compute System '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0452.json [hunk 20]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0452.json]: u'version' is a required property
[u'Cisco Tetration Analytics '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0421.json [hunk 21]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0421.json]: u'version' is a required property
[u'Cisco Prime Access Registrar '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0440.json [hunk 22]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0440.json]: u'version' is a required property
[u'Cisco Data Center Network Manager '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0434.json [hunk 23]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0434.json]: u'version' is a required property
[u'Cisco SD-WAN Solution '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0433.json [hunk 24]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0433.json]: u'version' is a required property
[u'Cisco SD-WAN Solution '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0438.json [hunk 25]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0438.json]: u'version' is a required property
[u'Cisco Umbrella '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0435.json [hunk 26]
```diff
@@ -1,18 +1,77 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0435.json]: u'version' is a required property
[u'Cisco Umbrella '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0454.json [hunk 27]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0454.json]: u'version' is a required property
[u'Cisco Cloud Services Platform 2100  '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0459.json [hunk 28]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0459.json]: u'version' is a required property
[u'Cisco Enterprise NFV Infrastructure Software '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0444.json [hunk 29]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0444.json]: u'version' is a required property
[u'Cisco Packaged Contact Center Enterprise '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0464.json [hunk 30]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0464.json]: u'version' is a required property
[u'Cisco Data Center Network Manager '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0463.json [hunk 31]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0463.json]: u'version' is a required property
[u'Cisco Network Services Orchestrator '] is not of type u'string'

File: 2018/0xxx/CVE-2018-0458.json [hunk 32]
```diff
@@ -1,18 +1,73 @@
 {
```
Comment (cve-team) [inline 2018/0xxx/CVE-2018-0458.json]: u'version' is a required property
[u'Cisco Prime Collaboration Assurance '] is not of type u'string'

**Pull request closed without merging**